### ISSUE TITLE
What's the path to access the web vault?

### ISSUE THREAD

**Issue by UntouchedWagons** [VictorNine/bitwarden-go#43]
I've got bitwarden-go installed in an lxc container and have it listening on 0.0.0.0:8000 but when I point my browser to 192.168.0.37:8000 it just says '404 page not found'

**Comment by korney4eg** [comment]
Hey @UntouchedWagons it  seems that it doesn't have web UI  (web vault),  so you  can only use API by bitwarden clients.

If you want web vault check out https://github.com/dani-garcia/bitwarden_rs it  seems that current project is not maintained anymore since april

**Comment by UntouchedWagons** [comment]
Yeah I've got that one installed in its own container, but I really don't like docker, I find it a pain to troubleshoot.

**Comment by korney4eg** [comment]
you can run it without docker. just Install golang, build application  and use it :)

**Comment by UntouchedWagons** [comment]
I mean Dani Garcia's implementation

**Comment by korney4eg** [comment]
Then you would need to install Rust. Belive me it takes soooooooo long to build this programm ;)
